internal/subscription/validation_test.go

```diff
@@ -445,3 +445,32 @@ func Test_validateFilterExpression(t *testing.T) {
 		assert.Equal(t, test.filterExpression, m.FilterExpression)
 	}
 }
+
+func Test_validatePushEndpoint(t *testing.T) {
+	ctx := context.Background()
+
+	type testCase struct {
+		inputConfig *metrov1.PushConfig
+		err         error
+	}
+
+	tests := [2]testCase{
```

Comment:
@subintp can we add another test case to test the InvalidPushEndpointURL error?

Reply:
Done